cameo/api/designer.py

```diff
@@ -73,7 +73,7 @@ def __call__(self, strategy):
 
 class _OptGeneRunner(_OptimizationRunner):
     def __call__(self, strategy):
-        max_evaluations = 20000
+        max_evaluations = 15000
```

Comment:
I agree with you. The next feature is going to be able to give a configuration file with this things. However, for the sake of publication, this is has much as I decided to change.